editor/editor_inspector.cpp

```diff
@@ -1094,25 +1105,7 @@ void EditorInspectorCategory::_notification(int p_what) {
 
 Control *EditorInspectorCategory::make_custom_tooltip(const String &p_text) const {
 	tooltip_text = p_text;
-	EditorHelpBit *help_bit = memnew(EditorHelpBit);
-	help_bit->add_theme_style_override("panel", get_theme_stylebox(SNAME("panel"), SNAME("TooltipPanel")));
```

Comment:
This was commented out in `EditorProperty`'s implementation, and I couldn't notice a difference between category (Node names) and property tooltips, so I removed it.